src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs

```diff
@@ -14,7 +14,8 @@ public record ExperimentalFeaturesEnabled(
     bool? SourceMapping,
     bool? ParamsFiles,
     bool? UserDefinedTypes,
-    bool? UserDefinedFunctions)
+    bool? UserDefinedFunctions,
+    bool? DynamicTypeLoadingEnabled)
```

Comment:
I think we can remove the `Enabled` suffix since it's already part of the record name.